lib/plugins/aws/package/compile/events/cognitoUserPool/index.js

```diff
@@ -32,9 +32,15 @@ class AwsCompileCognitoUserPoolEvents {
       'after:package:finalize': this.mergeWithCustomResources.bind(this),
     };
 
-    // TODO: Complete schema, see https://github.com/serverless/serverless/issues/8028
     this.serverless.configSchemaHandler.defineFunctionEvent('aws', 'cognitoUserPool', {
       type: 'object',
+      properties: {
+        pool: { type: 'string', maxLength: 128, regexp: /^[\w\s+=,.@-]+$/.toString() },
+        trigger: { type: 'string', enum: validTriggerSources },
```

Comment:
Isn't `type` superfulous when `enum` is declared?

Reply:
Yes it is, I updated accordingly :)